### PR TITLE
Translation into Russian. Buddy

### PR DESCRIPTION
**Pull request by kobakirill**
Translation into Russian. Buddy. Localization sections have been created

### CODE REVIEW

**Comment by kobakirill** [comment]
Test

**Comment by gagarinlg** [comment]
What are you doing? Open/close/open/...

**Comment by kobakirill** [comment]
> What are you doing? Open/close/open/...
sorry

**Comment by freshollie** [comment]
Nice! Thanks for these! I need to fix the e2e tests or this won't deploy so will merge this PR after that's fixed

**Review by freshollie — APPROVED** [review]

File: locales/languages.json [hunk 1]
```diff
@@ -1 +1 @@
-["en", "zh", "de", "es", "fr", "cs"]
+["en", "ru", "zh", "de", "es", "fr", "cs"]
```
Comment (rotorman) [inline locales/languages.json:1]: Ideally this list should be alphabetical, with the exception of universal en being first.

Comment (freshollie) [reply locales/languages.json:1]: Oops missed you review. Atm they are not alphabetical, but I would enforce this at rendering anyway if we wanted to? The order of the list here is not meaningful

**Comment by kalciferr** [comment]
I just wanted to get the Russian language in Edgetx, but I realized that this was impossible because I don’t understand anything here

**Comment by rotorman** [comment]
Adjust the radio model according to your need:
https://buddy.edgetx.org/#/flash?filters=includePrereleases&version=nightly&target=tx16s&selectedFlags=language%3DRU

**Comment by kalciferr** [comment]
I thought maybe there was also a Russian voice acting for edgetx.  Thank you

**Comment by rotorman** [comment]
Have a look at: https://github.com/EdgeTX/edgetx-sdcard-sounds/tree/main/SOUNDS/ru

**Pull request merged**